src/components/popover.vue

```diff
@@ -89,7 +89,8 @@ export default {
           html: true,
           placement: this.placement,
           trigger: 'manual',
-          animation: false
+          animation: false,
+          sanitize: false
```

Comment:
We do our own sanitization of user data (e.g., using `{{ ... }}` from Vue), so we don't need Bootstrap to do it for us. Bootstrap's [sanitizer](https://getbootstrap.com/docs/3.4/javascript/#js-sanitizer) removes a lot of elements that we want to show in hover cards, including `<dl>`, `<dt>`, `<dd>`, and `<time>`. We could tell Bootstrap to ignore those elements, but I really think we just don't need Bootstrap to be modifying the HTML we pass to it. When we replace the Bootstrap popover plugin with something else (#1084), we won't need the replacement to do any sanitization. We just need something to help us position the popover.